plugin.yaml

```diff
@@ -383,6 +383,11 @@ providers:
         name: publish
         title: 'publish'
         description: 'Publish a container port (eg xxxx:yyyy)'
+      - type: String
```

Comment:
Hi @TheLastProject, thank you for the PR 
the type here should be FreeSelect or Select. I think the best way is to use a FreeSelect with a default value (to not break current jobs)
Thanks

Reply:
Okay, done